geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GetMemberInformationFunction.java

```diff
@@ -142,12 +139,22 @@ public void execute(FunctionContext functionContext) {
           numConnections = numConnections + status.getNumberOfConnections();
         }
         memberInfo.setClientCount(numConnections);
+
+        ServerLauncher.ServerState state = ServerLauncher.getServerState();
+        if (state != null) {
```

Comment:
Are we not handling the null case? When the state is null, should we throw an error? Should we provide a default status?

@jdeppe-pivotal